help/destinations/catalog/advertising/liveramp-distribution.md

```diff
@@ -158,39 +189,117 @@ To configure details for the destination, fill in the fields below.
 
+To configure details for the destination, fill in the fields below.
+
+![Platform UI image showing the customer data fields for the iHeartMedia destination.](../../assets/catalog/advertising/liveramp-distribution/LR_iHeart_DestSpecific.png)
+
+* **[!UICONTROL Client Name]**: Your advertiser account name, as you would like it to be shown to the destination partner. Use your company name. Do not use spaces or special characters.
+
 ### [!DNL Index Exchange] {#index-exchange}
 
+>[!CONTEXTUALHELP]
+>id="platform_destinations_liveramp_distribution_index_advertiseraccountname"
+>title="Account Name"
+>abstract="Your Index Exchange client account name. Do not use spaces or special characters."
+
+To configure details for the destination, fill in the fields below.
+
+![Platform UI image showing the customer data fields for the Index Exchange destination.](../../assets/catalog/advertising/liveramp-distribution/LR_IndexExchange_DestSpecific.png)
+
+* **[!UICONTROL Account Name]**: Your Index Exchange client account name. Do not use spaces or special characters.
+
 ### [!DNL Magnite CTV Platform] {#magnite}
 
+>[!CONTEXTUALHELP]
+>id="platform_destinations_liveramp_distribution_magnitectv_client"
+>title="Client"
+>abstract="Your Client name, as you would like it to be shown to the destination partner. Use your company name. Do not use spaces or special characters."
+
+To configure details for the destination, fill in the fields below.
+
+![Platform UI image showing the customer data fields for the Magnite CTV destination.](../../assets/catalog/advertising/liveramp-distribution/LR_MagniteCTV_DestSpecific.png)
+
+* **[!UICONTROL Client]**: Your Client name, as you would like it to be shown to the destination partner. Use your company name. Do not use spaces or special characters.
+
 ### [!DNL Magnite DV+ (Rubicon Project)] {#magnite-dv}
 
-### [!DNL One Fox] {#fox}
+>[!CONTEXTUALHELP]
+>id="platform_destinations_liveramp_distribution_magnitedv+_partnerid"
+>title="PartnerId"
+>abstract="The Rubicon Project Partner ID associated with the publisher who owns the segment/data. Please contact your Rubicon Project account representative if unsure of the value you should use."
 
-### [!DNL Pandora] {#pandora}
+>[!CONTEXTUALHELP]
+>id="platform_destinations_liveramp_distribution_magnitedv+_seatid"
+>title="SeatId"
+>abstract="Magnite DV+ "seatId" provided by your Magnite account manager"
+
+To configure details for the destination, fill in the fields below.
+
+![Platform UI image showing the customer data fields for the Magnite DV+ destination.](../../assets/catalog/advertising/liveramp-distribution/LR_MagniteDV_DestSpecific.png)
+
+* **[!UICONTROL PartnerId]**: The Rubicon Project Partner ID associated with the publisher who owns the segment/data. Please contact your Rubicon Project account representative if unsure of the value you should use.
+* **[!UICONTROL SeatId]**: Magnite DV+ "seatId" provided by your Magnite account manager
```

Comment:
Please update these fields to Partner ID and Seat ID, in both the docs and the destination config.

Reply:
@bhealy21 I see you updated this in the destination config, but not in the documentation. Please update these fields to Seat ID and Partner ID.